Simplenote/src/main/java/com/automattic/simplenote/utils/ReviewAccountVerifier.java

```diff
@@ -0,0 +1,74 @@
+package com.automattic.simplenote.utils;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import com.automattic.simplenote.Simplenote;
+import com.automattic.simplenote.models.Account;
+import com.simperium.client.Bucket;
+import com.simperium.client.BucketObjectMissingException;
+
+import static com.automattic.simplenote.models.Account.KEY_EMAIL_VERIFICATION;
+
+/**
+ * When a user login into their account, Simplenote starts indexing the different buckets including Bucket<Account>.
+ * We setup a OnNetworkChangeListener on the account bucket to listen when the bucket is indexed. One of the parameters
+ * of the callback onNetworkChange called is Bucket.ChangeType. If this parameter is INDEX, then we know that the
+ * account bucket was indexed (synchronized). When the account bucket is synchronized, we can check whether the account
+ * was verified or not
+ */
+public class ReviewAccountVerifier implements Bucket.OnNetworkChangeListener<Account> {
```

Comment:
The name here is curious to me. I don't know if I'm missing something, but it seems like the purpose of this class is to watch for verification of an account and report that. To that end I might expect to see something like `AccountVerificationWatcher` or `AccountVerificationListener`

Any thoughts on the name or explanation?

Reply:
I changed it to `AccountVerificationWatcher`. Fixed in 79d4813bf85e462852237f08a6c4800f50a713d5.